src/XGBoost.jl

```diff
@@ -2,9 +2,23 @@ __precompile__()
 
 module XGBoost
 
-include("xgboost_lib.jl")
+using Compat
 
-export DMatrix, Booster
-export xgboost, predict, save, nfold_cv, slice, get_info, set_info, dump_model, importance
+include("libxgboost.jl")
+include("core.jl")
+include("training.jl")
+include("crossvalidation.jl")
+include("legacy.jl")
+include("callbacks.jl")
+
+export DMatrix
+export get_base_margin, get_float_info, get_label, get_uint_info, get_weight, num_col, num_row,
+    save_binary, set_base_margin, set_float_info, set_group, set_label, set_uint_info, set_weight
+
+export Booster
+export attr, attributes, boost, dump_model, eval, eval_set, get_dump, load_model, predict,
```

Comment:
Thanks! Fixed this by extending Core.eval since the module already implicitly calls `using Core`, thus popping a warning when using `import`.